corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -176,6 +176,14 @@ hqDefine("data_dictionary/js/data_dictionary", [
         self.showAll = ko.observable(false);
         self.availableDataTypes = typeChoices;
         self.fhirResourceTypes = ko.observableArray(fhirResourceTypes);
+
+        const params = new URLSearchParams(document.location.search);
+        self.showDeprecatedCaseTypes = ko.observable(params.get("load_deprecated_case_types") !== null);
+
+        // Elements with this class have a hidden class to hide them on page load. If we don't do this, then the elements
+        // will flash on the page for a bit while the KO bindings are being applied.
+        $(".deprecate-case-type").removeClass('hidden');
```

Comment:
A better way to achieve this would be to put a `ko-template` class on a parent div and then it will stay hidden until `koApplyBindings()` is done on the element. 

To do it in this manner will require significant refactoring however, since the entire HTML template is binded to `dataDictionaryModel`. We would essentially need to break up this model and do separate bindings to chunks of the HTML template.

TL:DR - Given the refactoring that is required to achieve a more elegant outcome, I will leave it as such for now.